be/src/exec/hdfs_scanner.cpp

```diff
@@ -242,6 +242,14 @@ Status HdfsScanner::open_random_access_file() {
     return Status::OK();
 }
 
+int64_t HdfsScanner::estimated_mem_usage() const {
+    if (_shared_buffered_input_stream == nullptr) {
+        // don't read data in columnar format, usually in a fixed size.
+        return 32 * 1024 * 1024;
```

Comment:
When will this situation happen?

Reply:
right now it's only csv format.

Reply:
maybe you can add a comment to explain it.

Reply:
OK.